proj/src/kmeans.cpp

```diff
@@ -2,6 +2,7 @@
 #include <vector>
 #include <string>
 #include <opencv2/opencv.hpp>
+#include <stdexcept>
 #include "kmeans.hpp"
 
 using namespace std;

```

Comment:
It is an absolute red light for any coder.  Never do this in front of a professional C++ developer.  It is a sure way to keep offers away.

Reply:
@yungyuc Sorry for late reply. What does this comment  mean? Do you mean that the code should not use "using namespace std" ?

Reply:
Correct.  No code that is available for others to read should `using namespace std;`